arch/arc/core/fatal.c

```diff
@@ -16,13 +16,14 @@
 #include <offsets_short.h>
 #include <toolchain.h>
 #include <arch/cpu.h>
-#include <logging/log_ctrl.h>
+#include <logging/log.h>
+LOG_MODULE_DECLARE(os);
```

Comment:
Don't this module need to be register at some point ?

Reply:
it is, in kernel/

Reply:
indeed, somehow I didn't see it :/ it is registered in init.c